### PR TITLE
Pricing FAQs

### PR DESCRIPTION
**Pull request by kennethsequeira**
image

### CODE REVIEW

File: erpnext_com/www/pricing/faq.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	<div class='container'>
+		<h1>Pricing FAQs</h1>
+        <p class="lead">Some common questions related to our services<p>
+   </div>     
+</section>
+
+##### Q. How is the fees calculated for the number of users?
+You only pay for the number of System users in your accounts. i.e those who can access the desk.
+
+##### Q. Who is a system user?
```
Comment (rmehta) [inline erpnext_com/www/pricing/faq.md]: lets explain this without using "Desk". We can say, system user is someone who uses core modules - not portals. Customers, Suppliers, Students, Guardians etc are not system users.

Desk is going to change in v12 (pretty soon)

File: erpnext_com/www/pricing/faq.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        alt="ERPNext Desk Screen" style="padding-top: 10px; padding-bottom: 17px; width:800px; margin:auto;">
+</div>
+
+You can pay for your subscription via Paypal or Credit card. If not, there's always Wire Transfer! Details for the same can be found <a href="https://erpnext.com/pricing/payment">here</a>
+
+##### Q. Do I need to have a Paypal Account for subscription?
+We use Paypal as our payment gateway to handle international subscriptions. You do not need to have a Paypal account for continuing with the subscription.
+
+##### Q. I am running out of storage space/need to add more users. How do I go about doing so?
+You can add more space/users from within your account. Just go to **My Account > Upgrade > Purchase Add-ons**.
```
Comment (rmehta) [inline erpnext_com/www/pricing/faq.md]: show an animated GIF here

**Comment by rmehta** [comment]
Please update with new pricing screenshot and merge!

**Comment by kennethsequeira** [comment]
- Rewrote the page in html (based on the partner FAQ page)
- Removed references of desk as requested
- Added gifs for upgrades and subscription

image

**Comment by rmehta** [comment]
@kennethsequeira payment page is redesigned!

**Comment by kennethsequeira** [comment]
@rmehta my bad. Took the screenshot with the old one. Here's the page with the redesigned payments page:

image

Here are the gifs added:

https://github.com/frappe/erpnext_com/blob/27b4af5f439dbd974b7d54a5cae7da2338a56d11/erpnext_com/public/images/subscribe.gif

https://github.com/frappe/erpnext_com/blob/27b4af5f439dbd974b7d54a5cae7da2338a56d11/erpnext_com/public/images/upgrade.gif

File: erpnext_com/www/pricing/faq.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+			</p>
+			<p class="mt-5">
+				<h5>I want to host ERPNext on my own server but need your support. Is this possible?</h5>
+				Yes! We do provide support for self-hosted instances as well. Check out our <a href="https://erpnext.com/support">support plan</a> for more details.
+			</p>
+			<p class="mt-5">
+				<h5>How to proceed with the subscription payment?</h5>
+                You can pay for your ERPNext account from the application itself! Go to <b>My Account > Upgrade</b>.
+                <div> <!--class="col-sm-12 text-center mar-t-40"> -->
+                    <img src="/assets/erpnext_com/images/subscribe.gif" class="img-responsive"
```
Comment (prasadarr) [inline erpnext_com/www/pricing/faq.html]: the GIFs here represent v11, can a gif on v12 be made?

Comment (kennethsequeira) [reply erpnext_com/www/pricing/faq.html]: There is no cloud account on v12 so far. Hence can't make a gif of the same

File: erpnext_com/www/pricing/faq.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                        alt="ERPNext Desk Screen" style="padding-top: 10px; padding-bottom: 17px; width:800px; margin:auto;">
+                </div>
+                You can pay for your subscription via Paypal or Credit card. If not, there's always Wire Transfer! Details for the same can be found <a href="https://erpnext.com/pricing/payment">here</a>                
+			</p>
+			<p class="mt-5">
+				<h5>Do I need to have a Paypal Account for subscription?</h5>
```
[the remark this answers is not in the document]
Comment (prasadarr) [reply erpnext_com/www/pricing/faq.html]: not done everywhere

**Pull request merged**